src/AppBundle/Issues/IssueListener.php

```diff
@@ -58,14 +58,39 @@ public function handleCommentAddedEvent($issueNumber, $comment)
      * Adds a "Needs Review" label to new PRs.
      *
      * @param int $prNumber The number of the PR
+     * @param int $prTitle  The title of the PR
+     * @param int $prBody   The full text description of the PR
      *
      * @return string The new status
      */
-    public function handlePullRequestCreatedEvent($prNumber)
+    public function handlePullRequestCreatedEvent($prNumber, $prTitle, $prBody)
     {
+        $prLabels = array();
+
+        // new PRs always require review
         $newStatus = Status::NEEDS_REVIEW;
+        $prLabels[] = $newStatus;
+
+        // the PR title usually indicates the affected component
+        if (preg_match('/^\[(?P<component>.*)\] .*/$', $prTitle, $matches)) {
+            $prLabels[] = $matches['component'];
+        }
+
+        // the PR body usually indicates if this is a Bug, Feature, BC Break or Deprecation
+        if (preg_match('/^\|\s*Bug fix?\s*\|\s*yes\s*$/', $prBody, $matches)) {
```

Comment:
This is my reasoning. If some user writes this by mistake:

```
| Bug fix? |yes
```

`\s*` will work but `\s+` won't. I can be wrong of course.